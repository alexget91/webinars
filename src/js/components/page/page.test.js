import React from "react";
import renderer from "react-test-renderer";
import {MemoryRouter} from "react-router-dom";
import Page from "./page";

it(`Page correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<MemoryRouter>
      <Page>
        <div>test</div>
      </Page>
    </MemoryRouter>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
