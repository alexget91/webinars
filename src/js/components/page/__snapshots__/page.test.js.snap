// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page correctly renders after relaunch 1`] = `
Array [
  <header
    className="page-header"
  >
    <a
      className="logo"
      href="/"
      onClick={[Function]}
    >
      <img
        alt="SEMrush"
        className="logo__image"
        src="test-file-stub"
        srcSet="test-file-stub 2x, test-file-stub 3x"
      />
    </a>
  </header>,
  <main
    className="page-main"
  >
    <div>
      test
    </div>
  </main>,
]
`;
